cvm-securekey-release-app/AttestationUtil.cpp

```diff
@@ -215,9 +216,70 @@ std::string Util::GetIMDSToken(std::string client_id)
 
     TRACE_OUT("Response: %s\n", responseStr.c_str());
 
+    json json_object = json::parse(responseStr.c_str());
+    std::string access_token = json_object["access_token"].get<std::string>();
+
+    TRACE_OUT("Access Token: %s\n", access_token.c_str());
+
     TRACE_OUT("Exiting Util::GetIMDSToken()");
 
-    return responseStr;
+    return access_token;
+}
+
+/// \copydoc Util::GetAADToken()
+std::string Util::GetAADToken()
+{
+    TRACE_OUT("Entering Util::GetAADToken()");
+
+    auto clientId = std::getenv("AKV_SKR_CLIENT_ID");

```

Comment:
The name of the env variables is not familiar. I think we need to align with the published AAD variables.
https://learn.microsoft.com/en-us/dotnet/api/azure.identity.environmentcredential?view=azure-dotnet

Reply:
AZURE_TENANT_ID, AZURE_CLIENT_ID, and AZURE_CLIENT_SECRET

Reply:
@canfikret Initially I thought about aligning to the Azure standard environment variables but then our use case requires a custom Service Principal just for AKV SKR access and if we reuse Azure env variables, it will not allow other Azure clients to connect using a different service principal for all other Azure resources

Reply:
@canfikret Did you get a chance to review this?